src/default_theme/section._

```diff
@@ -16,15 +16,14 @@
   <% } %>
 
   <%= md(section.description) %>
-
-  <div class='pre p1 fill-light mt0'><%= signature(section) %></div>
```

Comment:
Does this remove the function signature?

Reply:
It may be that I am not understanding the signature block in relation to
other output generated. Or, it could be there is another bug, or possibly
some missing field in the JSDoc comments I have?

At any rate it appeared redundant, so I took it out. Please see attached
screenshots for examples and let me know how you would like to proceed. If
this is "operator error" I would be interested in knowing the intent and
correct usage.

Reply:
I'm not seeing any screenshots attached. The function signature is an 'at a glance' way to see the function's inputs and outputs:

![2018-06-25 at 11 33 am](https://user-images.githubusercontent.com/32314/41868720-b4258442-786b-11e8-9a0c-47bbe5f970df.png)

You'll see the same pattern in [immutable docs](https://facebook.github.io/immutable-js/docs/#/), as well as many other documentation systems. In the most pedantic sense, yes - the signature repeats information elsewhere - but in terms of explaining how one calls a function, it's a very useful feature that shouldn't be removed.

Reply:
I see. I am guessing this is due to our declarations being constant assignments with arrow functions? We do not have any params in the showing up in the included div. 

I will go ahead and add that back and fix the typo.

![without_div](https://user-images.githubusercontent.com/40205008/41871721-b95c93de-788d-11e8-932c-32680a8a533b.png)
![with_div](https://user-images.githubusercontent.com/40205008/41871726-bbcc2a3a-788d-11e8-9ae7-d38f0ed09e1c.png)